examples/client.php

```diff
@@ -179,6 +208,27 @@ public function createSenderExample() {
         }
     }
 
+    public function getSenderByExternalIdExample() {
+        $sendersApi = new SendersApi();
+        $externalId = 'SENDER-2b59deff';
+        try {
+            $senderResponse = $sendersApi->getSenders(null, null, null, null, $externalId);
+            $sender = reset($senderResponse->getObject());
+            echo "Sender found! External ID: {$sender->getExternalID()}", PHP_EOL;
+            return $sender->getExternalId();
```

Comment:
Same with here. The sender should be returned here (as per the method name) rather than the external ID